### ISSUE TITLE
Can't browse images when selecting an image with a new document that has no Id yet

### ISSUE THREAD

**Issue by alevosia** [robinpyon/sanity-plugin-media#20]
Hi! Thank you for this awesome plugin. I've encountered an error when selecting an image with a new document whose Id is not yet initialized due no input/changes yet.

`{"error":{"description":"Unable to parse value of \"$documentId=undefined\". Please quote string values.","type":"httpBadRequest"}}`

The API request: 
```
'https://myprojectid.api.sanity.io/v1/data/query/dataset?query=\n' +
  '    {\n' +
  '      "items"%3A *[_type %3D%3D "sanity.imageAsset"] {\n' +
  '            _id%2C\n' +
  '            _updatedAt%2C\n' +
  '            extension%2C\n' +
  '            metadata {\n' +
  '              dimensions%2C\n' +
  '              isOpaque%2C\n' +
  '            }%2C\n' +
  '            originalFilename%2C\n' +
  '            size%2C\n' +
  '            url\n' +
  '          } | order(_updatedAt desc) [0...50]%2C\n' +
  '    }\n' +
  '  &%24documentId=undefined'
  ```
  
Wrapping `undefined` with double quotes sends the request successfully. 
  
I have to fill in another field in order to initialize the document's Id (turning it into a draft) before I can open the media library.

**Comment by robinpyon** [comment]
Thanks for reporting this @alevosia! This should be fixed in `1.0.1` – please reopen and let me know if you're still seeing it

**Comment by alevosia** [comment]
It's fixed, thanks! I just realized as well after forking and checking module. 😂Thanks!
